src/main/index.js

```diff
@@ -423,7 +427,7 @@ function runApp() {
   })
 
   app.on('activate', () => {
-    if (mainWindow === null) {
+    if (mainWindow == null) {
```

Comment:
Either use simplified checking
```js
if (mainWindow) {
```
Or use strict equality, even if it's more verbose
```js
if (mainWindow === null || mainWindow === undefined) {
```